sound/soc/sof/ipc4-loader.c

```diff
@@ -80,6 +80,12 @@ static ssize_t sof_ipc4_fw_parse_ext_man(struct snd_sof_dev *sdev,
 	dev_dbg(sdev->dev, "Header length: %u, module count: %u\n",
 		fw_header->len, fw_header->num_module_entries);
 
+	/* copy the fw_version into debugfs at first boot */
```

Comment:
@yongzhi1 you want to copy only the base_fw version here isnt it? If you happen to load module libraries during boot, this will get overwritten by the library version isnt it>?

Reply:
Hi, @ranj063 , that's correct, we are only interested in the base_fw version, as an alternative way to get the following info:
```
[    6.395430] sof-audio-pci-intel-mtl 0000:00:1f.3: Loaded firmware library: ADSPFW, version: 2.7.99.1
```

Reply:
then can you restrict this to base_fw only?

Reply:
Ack, will update in next version, thanks!!